tests/client/amo/components/TestSearchResults.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  renderIntoDocument as render,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+
+import SearchResults from 'amo/components/SearchResults';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+describe('<SearchResults />', () => {
+  function renderResults(props) {
+    return findRenderedComponentWithType(render(
+      <SearchResults i18n={getFakeI18nInst()} {...props} />
+    ), SearchResults).getWrappedInstance();
+  }
+
+  it('renders empty search results container', () => {
+    const root = renderResults();
+    assert.include(root.message.textContent, 'enter a search term');
+  });
+
+  it('renders no results when searched but nothing is found', () => {
+    const root = renderResults({
+      count: 0,
+      hasSearchParams: true,
+      loading: false,
+      results: [],
+    });
+    assert.include(root.message.textContent, 'No results were found');
+  });
+
+  it('renders error when no search params exist', () => {
+    const root = renderResults({ hasSearchParams: false });
+    assert.include(root.message.textContent, 'enter a search term');
+  });
+
+  it('renders error when no results and valid query', () => {
+    const root = renderResults({
+      filters: { query: 'test' },
+      hasSearchParams: true,
+    });
+    // Using textContent here since we want to see the text inside the p.
+    // Since it has dynamic content is wrapped in a span implicitly.
```

Comment:
I know you didn't change this but if you'd like this could likely use `root.message.children` now, react doesn't wrap in spans anymore and this comment can then go away.